frontend/src/lib/components/PropertyKeyInfo.tsx

```diff
@@ -189,6 +189,10 @@ export const KEY_MAPPING: KeyMappingInterface = {
             label: 'Pageview',
             description: 'When a user loads (or reloads) a page.',
         },
+        $pageview_id: {
+            label: 'Pageview ID',
+            description: 'The ID for the pageview event.',
```

Comment:
```suggestion
            description: 'The ID for a pageview event. All events from the web SDK have the pageview id from the preceding pageview.',
```

Reply:
I added this property and forgot about this. Does that explanation make sense?

Reply:
(I'm very confused by this property), what does the ID here mean? Do all events have IDs, and is this somehow more special?

Reply:
and since this will now be verified, so will show up higher, what does it mean to filter on pageview_id over pageview? As an end user I'm starting to get confused 😅 .

Reply:
Maybe it should be internal/hidden 🤔 

We added it as part of the performance requests work. Each time there is a pageview event in the web sdk we rotate an ID. That pageview event gets the ID as a property, and so does every event emitted from then until the next pageview. 

It was so we could say "these events were part of this pageview". Primarily to tie from a pageview to its network requests

Reply:
I see, makes sense, yeah, don't think users would ever do anything with this then?

Maybe this description makes sense then: "PostHog's internal ID for matching events to a pageview" - tells people not to use it, shows up when filtering (slightly annoying), but at the same time, doesn't hide it from view since that will be harder than a description change lol.

Reply:
```suggestion
            description: 'PostHog's internal ID for matching events to a pageview.',
```

Reply:
Good to know this, in that case we should definitely mark these as system properties (which are hidden by default). Made that change